packages/@react-spectrum/s2/src/Field.tsx

```diff
@@ -76,6 +76,7 @@ export const FieldLabel = forwardRef(function FieldLabel(props: FieldLabelProps,
       className={style({
         gridArea: 'label',
         display: 'inline',
+        whiteSpace: 'nowrap',
```

Comment:
Doesn't this break the long label example in the form story? Looks like it overflows now. I think we do want the label to wrap, just not right before the contextual help. We could potentially use the same solution we used for the necessity indicator, where we wrapped the contextual help in a `<span>` with `white-space: nowrap` and put a `&nbsp;` before it (instead of the margin). That should allow the label to wrap, but prevent the contextual help from being on a line by itself.

Reply:
Makes sense, updated